aries-site/src/pages/templates/inline-notifications.mdx

```diff
@@ -0,0 +1,143 @@
+| **5** | **Close button**             | Dismisses the notification                                   |                                             |                                                                                                                        |
+
+## Placement
+
+Inline notifications should be placed near the content to which they are
+related.
+
+- If the notification is related to a specific element or section of content,
+  it should be placed near that element or section of content.
+- If the notification is related to the entire page, it should be placed near
+  the top of the page.
+- In forms, notifications may be placed either at the top or bottom, depending
+  on the context. For example, a notification providing form-level
+  validation should be placed at the bottom of the form between the last input
+  and submit button. A notification indicating a new option or configuration is
+  available would be appropriate to place at the top of the form.
+
+## Format
+
+- Titles and messages should be brief and clear.
+- Messages should generally be no longer than two lines.
+- Longer messages should wrap on narrower screens; they should not be truncated.
+- Anchor links may be used to provide additional information about the
+  notification. The links should immediately follow the message and should be
+  limited to no more than two links.
+
+## Content guidelines
+
+Notifications reassure users that they are moving towards their goal. They should
+be relevant, timely, and informative providing assurance that their intended
+actions have been completed successfully. Or, if there is an issue, they should
+provide clear direction on how to resolve the issue.
+
+- **Relevant** &ndash; Notifications should be relevant to the user's current
+  task.
+- **Timely** &ndash; Deliver notifications in a timely manner and ensure they
+  are kept up to date with current status.
+- **Informative** &ndash; Messages shoud provide clear context and direction on
+  how to resolve any issues. Follow the [voice and tone guidelines](/guidelines/voice-and-tone) for writing
+  crisp and effective content.
+
+## Behavior
+
+- Never sticky and should scroll when there is other content.
+- The width and resizing behavior of an inline notification should scale alongside
+  related content. For example, when placed below the [PageHeader](/components/pageheader),
+  the inline notification should have the same width and responsive behavior as the
+  PageHeader.
+- Inline Notifications are not clickable. Instead use anchors to direct users to
+  take further action or to view the issue in full detail.
+- Do not place buttons within an inline notification.
+
+## Do's and dont's
+
+## Use cases
+
+### Status updates
+
+Inline notifications may be used to display okay, warning, and critical information.
```

Comment:
```suggestion
Inline notifications may be used to display normal, warning, and critical information.
```

We're currently using the terminology 'normal' instead of 'okay' within our status indicators guidance.

Reply:
👍

Reply:
Done.